persistence/in-memory.md

```diff
@@ -16,3 +16,5 @@ snippet: ConfiguringInMemory
 DANGER: All information stored in the In-Memory persistence is discarded when the process ends.
 
 NOTE: The [Delayed Retries](/nservicebus/recoverability/#delayed-retries) mechanism uses the [timeout manager](/nservicebus/messaging/timeout-manager.md) when a transport does not natively support delayed delivery. As Delayed Retries are enabled by default, using In-Memory persistence with a transport that uses the timeout manager has the risk of losing messages that have failed processing and are waiting for another retry attempt. Use In-Memory persistence only in scenarios where it is OK to lose messages.
+
+include: gatewaydedupe
```

Comment:
I think this should be `partial` instead of `include`

Reply:
```suggestion
partial: gatewaydedupe
```